src/core/coreTypes.ts

```diff
@@ -1,50 +1,33 @@
-import { IExpressRequest, IExpressResponse, IResponse, IRequest, HttpMethod } from "./types"
-import { NextFunction } from "../../example/node_modules/@types/express"
-
 // middleware type?
-export type MiddlewareFunction = (
-  ...middleware: Handler[]
-) => {
-  middleware: Handler[]
-  subendpoints: SubendpointsFunction
-  endpoints: EndpointsFunction
-}
-
-export type SubendpointsFunction = (
-  ...subendpoints: EndpointsBuilderResult[]
-) => {
-  middleware: Handler[]
-  subendpoints: EndpointsBuilderResult[]
-  endpoints: EndpointsFunction
-}
-
-export type EndpointsFunction = (endpoints: Record<string, EndpointMethod<unknown, any>>) => EndpointsBuilderResult
-
+export type MiddlewareFunction = (...middleware: Handler[]) => {
+    middleware: Handler[];
+    subendpoints: SubendpointsFunction;
+    endpoints: EndpointsFunction;
+};
+export type SubendpointsFunction = (...subendpoints: EndpointsBuilderResult[]) => {
+    middleware: Handler[];
```

Comment:
I'm guessing these are linted from different rule, can we keep it consistent with .eslintrc? are there any rules we changeup with in eslint & prettier?

Reply:
I'm using eslint for a my linter so it should be reading off `.eslintrc`, but I see a `.prettierrc.js` file too (prettier can just be installed using `npm install` right?). I just ran `eslint` and the formatting when back to what it was before this commit... not sure what happened earlier. I think VSCode might have been running its own linter on all the files in the background.